### PR TITLE
Document how to fix a published release

### PR DESCRIPTION
**Pull request by zcbenz**

### CODE REVIEW

File: docs/development/releasing.md [hunk 1]
```diff
@@ -230,4 +230,34 @@ Then edit the release on GitGub:
...
 1. Remove `beta` from the release name: electron v1.7.5 ~~beta~~
 1. Uncheck the `prerelease` checkbox.
-1. Click "Update release" 
\ No newline at end of file
+1. Click "Update release"
+
+## Fix missing binaries of a release manually
+
+In the case of a corrupted release with broken CI machines, we might have to
+re-upload the binaries for a already published release.
```
Comment (felixrieseberg) [inline docs/development/releasing.md]: `an already published`

File: docs/development/releasing.md [hunk 2]
```diff
@@ -230,4 +230,34 @@ Then edit the release on GitGub:
...
+
+The first step is to go to the Releases page and delete the corrupted binaries
```
Comment (felixrieseberg) [inline docs/development/releasing.md]: `The first step is to go to the [Releases](https://github.com/electron/electron/releases) page`

**Comment by zcbenz** [comment]
> Does this line mean the command should be invoked with just one of those architectures specified? Or is this shell magic for specifying all three at once?

Only one arch can be passed.

**Comment by zeke** [comment]
Thanks @zcbenz ⭐️

**Pull request merged**